tests/suite/visualize/square.typ

```diff
@@ -144,3 +144,9 @@
 // Test that square sets correct base for its content.
 #set page(height: 80pt)
 #square(width: 40%, rect(width: 60%, height: 80%))
+
+--- square-height ---
```

Comment:
Imo there should be more tests, in particular for circle and maybe (at least one of) rect / ellipse (even though those should not exhibit buggy behavior in principle).

Also add a test for missing height (instead of missing width).